src/components/table/table.tsx

```diff
@@ -175,6 +177,35 @@ export class Table {
         this.tabulator.setColumns(this.getColumnDefinitions());
     }
 
+    /*
+     * Tabulator requires that the html element it's rendered inside
+     * has a size before it's created, otherwise it doesn't consider
+     * it self renderedy completely. (the callback "renderComplete"
+     * is never run).
+     *
+     * @param table {HTMLElement}
+     * @param options {Tabulator.Options}
+     *
+     * @returns {void}
+     */
+    private initTabulatorComponent(
+        table: HTMLElement,
+        options: Tabulator.Options
+    ) {
+        // Some browsers does not implement the ResizeObserver API...
+        // If that's the case lets just create the table no
+        // matter if its rendered or not.
+        if (!('ResizeObserver' in window)) {
+            this.tabulator = new TabulatorTable(table, options);
+        }
```

Comment:
Should we add `return; ` to avoid the error logs.

Reply:
It means we are going to have a broken table if the browsers don't implement the ResizeObserver API?

Reply:
You are right!

Yes but it works after you have sorted the table and the problem doesn't seem to appear in the webclient on iphone :)